packages/support/test/logger/helpers.js

```diff
@@ -3,10 +3,12 @@ import _ from 'lodash';
 import { logger } from '../../lib';
 
 
+let sandbox;
 
 function setupWriters () {
-  return {'stdout': sinon.spy(process.stdout, 'write'),
-          'stderr': sinon.spy(process.stderr, 'write')};
+  sandbox = sinon.createSandbox();
+  return {'stdout': sandbox.spy(process.stdout, 'write'),
```

Comment:
might as well update this line to remove unnecessary quotes around keys; i thought we had a lint rule for this